frontend/components/textShapeToSound/TextShapeAnalysis.js

```diff
@@ -104,83 +104,81 @@ const TextShapeAnalysis = () => {
             });
     };
 
+    const styles = {
```

Comment:
Since this is just getting used once, we could inline it into the `style` attribute where it gets used in `makeTextInput`.